spec/lib/api/v3/utilities/path_helper_spec.rb

```diff
@@ -374,71 +319,55 @@
     describe '#work_packages' do
```

Comment:
The `shared_example` above this line should now be deletable, since you replaced all its usages.